doc/api/child_process.md

```diff
@@ -33,6 +33,14 @@ process will block waiting for the pipe buffer to accept more data. This is
 identical to the behavior of pipes in the shell. Use the `{ stdio: 'ignore' }`
 option if the output will not be consumed.
 
+The command lookup will be performed using `options.env.PATH` environment
```

Comment:
So this is the thing I wasn't sure about in https://github.com/nodejs/node/issues/20605#issuecomment-388913376. On Windows it is true that the command lookup works that way but I don't know about the other platforms (there's no equivalent libuv code so it's a question of how the Posix system calls behave).

Reply:
Nit sure where the code is but it behaves this way.

```cmd
 ➔ dev/node/node uname -a                                                                                                                                                 
Linux zou 5.5.6-arch1-1 #1 SMP PREEMPT Mon, 24 Feb 2020 12:20:16 +0000 x86_64 GNU/Linux
```
Node.js 13.9.0:
![Screenshot from 2020-03-04 22-16-50](https://user-images.githubusercontent.com/9109612/75919293-f1d1fc80-5e65-11ea-8b4a-e075540a2045.png)

Node.js 12.13.0
![Screenshot from 2020-03-04 22-18-09](https://user-images.githubusercontent.com/9109612/75919361-11692500-5e66-11ea-89f4-19f907f103af.png)

Empty `PATH` leads to `ENOENT`.